app/models/intermediate_evaluation.rb

```diff
@@ -3,13 +3,18 @@ class IntermediateEvaluation < ApplicationRecord
   belongs_to :group
 
   validates :starting_month, :ending_month, presence: true
+
   validate :month_period
 
+  delegate :name, :year, to: :group, prefix: true
+
   private
 
   def month_period
-    if starting_month.month > ending_month.month
-      errors.add(:starting_month, I18n.t('intermediate_evalution.errors.starting_month'))
+    if starting_month.to_s != '' && ending_month.to_s != ''
+      if starting_month.month > ending_month.month
```

Comment:
Acá para evitar ifs anidados podés usar lo que se llama safe operator: https://stackoverflow.com/questions/36812647/what-does-ampersand-dot-mean-in-ruby